website/assets/js/pages/articles/articles.page.js

```diff
@@ -41,15 +41,15 @@ parasails.registerPage('articles', {
           break;
         case 'guides':
           this.articleCategory = 'Guides';
-          this.categoryDescription = 'Learn more about how to deploy and use Fleet.';
+          this.categoryDescription = 'Learn more about how to use Fleet to accomplish your goals.';
           break;
         case 'announcements':
           this.articleCategory = 'Announcements';
           this.categoryDescription = 'The latest news from Fleet.';
           break;
-        case 'use-cases':
+        case 'deploy':
           this.articleCategory = 'Product';
```

Comment:
@eashaw, I think this should read Deploy? Does this need to match the actual category name? For this category landpage it would be better if the title read "Deployment guides."

Reply:
Like this, for example:
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/78363703/179646362-e937f375-5d01-46ba-a377-66f86512ffcc.png">

Reply:
I've updated the category name to be "Deployment guides"